src/Amazon.AspNetCore.Identity.Cognito/Amazon.AspNetCore.Identity.Cognito.csproj

```diff
@@ -16,22 +16,22 @@
     <RepositoryUrl>https://github.com/aws/aws-aspnet-cognito-identity-provider/</RepositoryUrl>
     <Company>Amazon Web Services</Company>
     <SignAssembly>true</SignAssembly>
-    <AssemblyVersion>2.2.0</AssemblyVersion>
-    <FileVersion>2.2.0</FileVersion>
+    <AssemblyVersion>3.0.0</AssemblyVersion>
+    <FileVersion>3.0.0</FileVersion>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
       <DefineConstants>NETSTANDARD_2_0</DefineConstants>
     </PropertyGroup>
-    <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
-      <DefineConstants>NETCOREAPP_3_0</DefineConstants>
+    <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+      <DefineConstants>NETCOREAPP_3_1</DefineConstants>
```

Comment:
"NETCOREAPP3_1" should already be defined so that we do not need to define our own: https://docs.microsoft.com/en-us/dotnet/standard/frameworks#how-to-specify-a-target-framework